aasemble/django/apps/mirrorsvc/models.py

```diff
@@ -160,3 +161,9 @@ def save(self, *args, **kwargs):
     def perform_snapshot(self):
         self.sync_dists()
         self.symlink_pool()
+
+
+class MirrorDefinitionForm(ModelForm):
+    class Meta:
+        model = Mirror
+        fields = ['url', 'series', 'components', 'public']
```

Comment:
I don't think the form belongs in models.py. Move it to a new forms.py or views.py (your choice).

I know there's another Form in buildsvc's models.py, but that's also a mistake. I'll move it somewhere else.